app/[locale]/academics/scholarships/page.tsx

```diff
@@ -1,9 +1,235 @@
-import WorkInProgress from '~/components/work-in-progress';
+import Link from 'next/link';
+import { ReactNode } from 'react';
 
-export default function Scholarships({
+import Heading from '~/components/heading';
+import ImageHeader from '~/components/image-header';
+import { buttonVariants } from '~/components/ui';
+import { getTranslations } from '~/i18n/translations';
+import { cn } from '~/lib/utils';
+
+export default async function Scholarships({
   params: { locale },
 }: {
   params: { locale: string };
 }) {
-  return <WorkInProgress locale={locale} />;
+  const text = (await getTranslations(locale)).Scholarships;
+
+  return (
+    <>
+      <ImageHeader
+        title={'Scholarships'}
+        headings={[
+          {
+            // National Scholarship Portal
+            label: 'NSP',
+            href: '#nsp',
+          },
+          {
+            // Har-Chhatravratti Scholarship
+            label: 'HCS (Haryana)',
+            href: '#hcs',
+          },
+          {
+            // Single Sign On (SSO) Scholarship
+            label: 'SSO (Rajasthan)',
+            href: '#sso',
+          },
+          {
+            // Mukhyamantri Medhavi Vidhyarthi Yojna (MMVY) (Madhya Pradesh)
+            label: 'MMVY (Madhya Pradesh)',
+            href: '#mmvy',
+          },
+          {
+            // Uttar Pradesh Scholarship
+            label: 'UPS',
+            href: '#ups',
+          },
+          {
+            // Post Matric Bihar Scholarship
+            label: 'PMBS',
+            href: '#pmbs',
+          },
+          {
+            // Prime Ministers Special Scholarship Scheme
+            label: 'PMSSS to J&K Students',
+            href: '#pmsss',
+          },
+        ]}
+        src={`https://nitkkr.ac.in/wp-content/uploads/2022/12/IMG_0496-scaled.jpg`}
```

Comment:
If we're just putting strings, we will not use this syntax.
```suggestion
        src="https://nitkkr.ac.in/wp-content/uploads/2022/12/IMG_0496-scaled.jpg"
```